### PR TITLE
Update graphics inspector section to use ObservedListModel to catch c…

### PR DESCRIPTION
**Pull request by KRLango**
…hanges to the graphics list

#1210

### CODE REVIEW

**Comment by KRLango** [comment]
I think #1219 should also be fixed by this

File: nion/swift/Inspector.py [hunk 1]
```diff
@@ -3195,12 +3195,14 @@ def __create_position_and_shape_ui(self) -> Declarative.UIDescriptionResult:
...
 
 
-class GraphicsCalibrationHandler(Declarative.Handler):
-    def __init__(self, document_controller: DocumentController.DocumentController, display_item: DisplayItem.DisplayItem):
-        super().__init__()
-
+class GraphicsSectionHandler(Declarative.Handler):
+    def __init__(self, document_controller: DocumentController.DocumentController,
+                 display_item: DisplayItem.DisplayItem,
+                 graphics_model: ListModel.ObservedListModel[DisplayItem.DisplayLayer]):
```
Comment (cmeyer) [inline nion/swift/Inspector.py:3201]: This needs a `super().__init__()` to properly initialize `Declarative.Handler`, which itself has to initialize `Observable`.

**Review by cmeyer — COMMENTED** [review]
Other than the missing `super.__init__()`, this looks good. Any reason it is still a draft?

I'm not sure why mypy doesn't catch the missing super.init... but see [this issue](https://github.com/python/mypy/issues/11809).

Also, note that `flake8 --ignore E501 nion/swift/Inspector.py` does not catch this issue either.

However, `pylint nion/swift/Inspector.py` it shows up as "nion/swift/Inspector.py:3199:4: W0231: __init__ method from base class 'Handler' is not called (super-init-not-called)". However it is mixed into 100's of other lint errors. Oh well.

**Review by lisham2000 — APPROVED** [review]
All looks good

**Review by Ion-e — COMMENTED** [review]
Looks good to me!

**Comment by cmeyer** [comment]
Squashed/merged here 2d73324c190d6a05b097286cac3a98a816ab6b64

**Pull request closed without merging**